scripts/test-external-partner-project.js

```diff
@@ -93,7 +93,7 @@ try {
   );
 }
 
-const useYarn = fs.existsSync(path.join(projectTempDir, 'yarn.lock'));
+const usePnpm = fs.existsSync(path.join(projectTempDir, 'yarn.lock'));
```

Comment:
~~I think we need a use yarn|pnpm|npm check now vs just yarn|npm.~~

Lets see how they run first